### PR TITLE
 Remove spar link from travel page scout list [issue #60] 

### PR DESCRIPTION
**Pull request by S1lentcodes**

### CODE REVIEW

File: scoutArea.php [hunk 1]
```diff
@@ -92,9 +92,14 @@ function scoutArea($in_existing_table = false) {
 				}
 				else if($user['location'] == $player->location && $user['user_id'] != $player->user_id) {
 					// Attack
-					echo "<a href='{$system->links['spar']}}&challenge={$user['user_id']}'>Spar</a>";
+					if($show_spar_link) {
+						echo "<a href='{$system->links['spar']}}&challenge={$user['user_id']}'>Spar</a>";
+					}
```
Comment (levimeahan) [inline scoutArea.php:97]:
```suggestion
					echo "<a href='{$system->links['spar']}}&challenge={$user['user_id']}'>Spar</a>";

```

Comment (levimeahan) [inline scoutArea.php:97]:
```suggestion
					echo "<a href='{$system->links['spar']}}&challenge={$user['user_id']}'>Spar</a>";

```

File: scoutArea.php [hunk 2]
```diff
@@ -8,7 +8,7 @@
 Algorithm:	See master_plan.html
 */
 
-function scoutArea($in_existing_table = false) {
+function scoutArea($in_existing_table = false, $show_spar_link = true) {
```
Comment (levimeahan) [inline scoutArea.php:11]:
```suggestion
function scoutArea($in_existing_table = false) {

```

**Comment by levimeahan** [comment]
Oh man I missed this one and did the change myself. Thanks for fixing all the warning messages here! Can you revert the scoutArea changes and rebase on master? Also what's your SC username so I can credit you the AK

**Comment by S1lentcodes** [comment]
Reverted scountArea and rebased. I am Silent on SC.

File: privateMessages.php [hunk 3]
```diff
@@ -472,7 +472,7 @@ function privateMessages() {
 
 	$Messaging->display('options');
 
-	switch($_GET['page']) {
+	switch((isset($_GET['page']) ? $_GET['page'] : '')) {
```
Comment (levimeahan) [inline privateMessages.php:475]:
```suggestion
	switch($_GET['page'] ?? '') {
```

**Review by levimeahan — APPROVED** [review]

**Pull request merged**